seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java

```diff
@@ -78,6 +82,8 @@ public static final class Builder {
 
         private final Map<String, Supplier<String>> headerSuppliers = new HashMap<>();
 
+        private int timeout = 0;
```

Comment:
can we initialize the timeout to be 60 seconds

Reply:
Yep; done.